.github/workflows/Coverage.yaml

```diff
@@ -0,0 +1,38 @@
+name: Coverage
+
+on:
+  pull_request:
+
+jobs:
+  coverage:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 11
+
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+
+      - name: Setup diff_cover
+        run: pip install diff_cover==7.7.0
```

Comment:
Went down a rabbit hole of trying to pin to a hash here and decided to timebox it (need to run to an interview).

I'm not sure how much SHA pinning here matters if these are public repos.

Reply:
Should I even be pinning a version here?  Could do `>7,<8`.

Reply:
thanks for spending your time saving us time! 💛